pmg/views.py

```diff
@@ -234,10 +235,16 @@ def bill(bill_id):
         'introduced': 1,
     }
     history = bill_history(bill)
+
+    if bill['status']:
```

Comment:
so if it doesn't exist in the dict, or it's None or empty string, you'll go to the else clause